Web/JS/Assignment-1/Js/Registration.js

```diff
@@ -0,0 +1,83 @@
+
+//Validation for Activating The Submit Button
+
+function activateSubmit() 
+{
+
+	if(document.getElementById("acceptCondition").checked == true)
+	{
+		document.getElementById("submitButton").disabled = false;
+		
+	}
+	else 
+	{
+		document.getElementById("submitButton").disabled = true;
+	}
+}
+
+// Validation for Checking all the conditions
+
+function validate()
+ {
+	var count=0;
+	// First name Validation
+	if(document.getElementById("firstName").value.trim() == "")
+	{
+		count=1;
```

Comment:
For validation error do not use alert as displaying so much alert is irritating for user. Display errors below fields.